uid-pages/page-admin-user-list/src/adminUserListBonitaV1.json

```diff
@@ -3075,19 +3187,26 @@
         "return {",
         "    statusCode: \"\",",
         "    failedCreationResponse: undefined,",
-        "    creationPayload: undefined",
+        "    creationPayload: undefined,",
+        "    hasRecentlyBeenCreated: false",
         "};"
       ],
       "exposed" : false
     },
     "refreshListHandler" : {
       "type" : "expression",
       "value" : [
-        "// After a user is created or updated",
-        "if ($data.changeStatusData.statusCode === 200 || $data.createUserData.statusCode === 200) {",
+        "// After the user is created",
+        "if ($data.createUserData.statusCode !== '' && $data.createUserData.hasRecentlyBeenCreated === false) {",
+        "    $data.userListData.timestamp = new Date().getTime();",
+        "    $data.createUserData.hasRecentlyBeenCreated = true;",
+        "}",
+        "",
+        "// After the status is changed",
+        "if ($data.changeStatusData.statusCode !== '' && $data.changeStatusData.hasRecentlyBeenActivated === false && $data.changeStatusData.hasRecentlyBeenDeactivated === false) {",
         "    $data.userListData.timestamp = new Date().getTime();",
-        "    $data.createUserData.statusCode = '';",
-        "    $data.changeStatusData.statusCode = '';",
+        "    $data.changeStatusData.hasRecentlyBeenActivated = true;",
+        "    $data.changeStatusData.hasRecentlyBeenDeactivated === true;",
```

Comment:
This seems weird, why are we verifying that hasRecentlyBeenDeactivated is equal to true?

Reply:
Done